portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -12,6 +12,8 @@ definition {
 		TestCase.setUpPortalInstance();
 
 		User.firstLoginPG();
+
+		JSONGroup.addGroup(groupName = "Test Site Name");
```

Comment:
Change to `WYSIWYG Site Name` to make it unique from other testcases. Want to avoid possible pollution from other testcases that may fail and are mixed into the same jenkins portal instance.